.github/workflows/deploy.yml

```diff
@@ -23,6 +19,9 @@ jobs:
     secrets: inherit
```

Comment:
There is some concern that deploying concurrently to Dockerhub could cause concerns. I'd be in favour of adding:

```yaml
 concurrency:
      group: ${{ github.event.repository.name }}-deploy-dockerhub
      cancel-in-progress: false
```

To eliminate this worry and then trying this out. It should be miles better than what we have.

Reply:
Update has been made.